src/main/kotlin/com/workos/user_management/models/AuthenticationFactors.kt

```diff
@@ -0,0 +1,20 @@
+package com.workos.user_management.models
+
+import com.fasterxml.jackson.annotation.JsonCreator
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.workos.common.models.ListMetadata
+
+/**
+ * A list of [AuthenticationFactor]s
```

Comment:
```suggestion
 * A list of authentication Factors.
```

Reply:
👍

Reply:
Oh, wait. Do these brackets actually mean something as far as these doc comments go? As in, would they link to the type-definitions of the relevantly named object if say a doc generator to HTML was run?

I had assumed these were copy-paste mistakes, but now realizing that may have been wrong.

Reply:
Yes. They also provide the full documentation of the type when hovering over in IntelliJ, which can be useful in those cases were the structure of the class is not trivial:

<img width="647" alt="image" src="https://github.com/workos/workos-kotlin/assets/165715960/40d44c91-95a9-40d6-b0f3-d7c0060ce836">

While it is not a big deal at the moment, as we don't rely on any autogened documentation for the SDKs yet, I went ahead and edited the documentation of the non-literal types found in the model classes.